index.js

```diff
@@ -121,7 +121,10 @@ class DockerExecutor extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        const [buildImage, buildTag] = config.container.split(':');
+        const containerName = config.container.split(':');
+
+        const buildTag = (containerName.length === 1) ? 'latest' : containerName.pop();
```

Comment:
I think it doesn't work with this image name: `docker-registry.foo.bar:5000/image/name`. The tag would be `5000/image/name` in this case. Need to check the form of the image name and switch the logic?